### PR TITLE
adds in language about who is eligible to join

### PR DESCRIPTION
**Pull request by TJkrusinski**
cursory statements stating we are open to you joining

### CODE REVIEW

**Comment by Robdel12** [comment]
:+1: Looks good to me

**Comment by scifisamurai** [comment]
:+1: Looks good to me as well :smiley:

File: readme.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ The Slack is invitation-only, but invitations are freely granted. You can reques
 
 Orlando Developers Slack was created to foster the software development community in Orlando. We encourage sharing your knowledge, asking questions, forming relationships and a general ambition to strengthen the community.
 
+### Am I eligible to join?
+
+In short, yes! We are really open to having as many people that want to be a part, be a part. There are no hard rules on who is able to join and who is not. The only consideration is that the puropose of the group is to grow local relationships. That means if have attended meet ups in Orlando, work for a company based out of Orlando, work with other developers in Orlando, or have some similar affiliation, join us!
```
Comment (sergiocruz) [inline readme.md]: "puropose" => "purpose" :wink:

**Comment by sergiocruz** [comment]
@TJkrusinski copy looks great! I just noticed what I believe to be a spelling mistake. Other than that we're :+1:

File: readme.md (hunk 1 above)
Comment (bdougie) [inline readme.md]: Maybe I am reading this weird, but is "be a part" suppose to be repeated?

**Comment by TJkrusinski** [comment]
Typo was amended, thanks @sergiocruz

**Comment by TJkrusinski** [comment]
force push github cache invalidation fail :(

**Pull request merged**